src/lib/rules/performance-budget/connections.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * @fileoverview Loads the connections.ini and updates it if needed
+ */
+
+import { readFile } from '../../utils/misc';
+import { NetworkConfig } from './types';
+
+
+/**
+ * Parses a string that contains a network configuration.
+ *
+ * The configuration string should have the following format:
+ *
+ * ```ini
+ * [Cable]
+ * label="Cable (5/1 Mbps 28ms RTT)"
+ * bwIn=5000000
+ * bwOut=1000000
+ * latency=28
+ * plr=0
+ * ``` *
+ */
+const parseConnection = (configText: string): NetworkConfig => {
+    const lines = configText.split('\n');
+    const config: NetworkConfig = {
+        bwIn: 0,
+        bwOut: 0,
+        id: '',
+        label: '',
+        latency: 0,
+        plr: 0
+    };
+
+    lines.forEach((line) => {
+        const [key, value] = line.trim().split('=');
+
+        if (key.startsWith('[')) {
+            config.id = key.replace(/[[\]]/g, '');
+
+            return;
+        }
+
+        config[key] = value;
```

Comment:
The type of the `bwIn`, `bwOut` and `latency` values are `string` instead of `number`. Maybe `parseInt` should be used to wrap around the `value` for those keys?

Reply:
Good catch!